README.md

```diff
@@ -140,11 +140,11 @@ By contributing to the project, you agree to the license and copyright terms the
 
 ### User documentation
 
-The latest documentation for OpenVINO™ Toolkit is available [here](https://docs.openvino.ai/). This documentation contains detailed information about all OpenVINO components and provides all important information which could be needed if you create an application which is based on binary OpenVINO distribution or own OpenVINO version without source code modification.
+The latest documentation for OpenVINO™ Toolkit is available [here](https://docs.openvino.ai/). This documentation contains detailed information about all OpenVINO components and provides all the important information you may need to create an application based on binary OpenVINO distribution or own OpenVINO version without source code modification.
 
 ### Developer documentation
 
-[Developer documentation](./docs/dev/index.md) contains information about architectural decisions which are applied inside the OpenVINO components. This documentation has all necessary information which could be needed in order to contribute to OpenVINO.
+[Developer documentation](./docs/dev/index.md) contains information about architectural decisions which are applied inside the OpenVINO components. This documentation provides all the necessary information you may need to contribute to OpenVINO.
```

Comment:
@ilya-lavrenov @ilyachur this is information that is relevant if you want to modify OpenVINO, even if just for yourself. For example: the conditional compilation document is targeted at end users, not specifically people who want to contribute to OpenVINO. Can we remove the "This documentation provides all the necessary information you may need to contribute to OpenVINO." sentence here to prevent giving the wrong impression? Might also be good to expand the first sentence a bit because the dev docs contain more than architectural decisions.

Reply:
I agree. 
My vision is that uses who build OpenVINO using different ways including sophisticated conditional compilation - are still users, while developers are those who change OV code.

Reply:
I appreciate your input. I will revert the sentence back to original.

Reply:
```suggestion
[Developer documentation](./docs/dev/index.md) contains information about architectural decisions which are applied inside the OpenVINO components.
```

Reply:
@helena-intel , @ilya-lavrenov what is your take on the suggestion above?

Reply:
CC @ilyachur

Reply:
@sgolebiewski-intel It's not really true. Basic information about the process of building OpenVINO is in the wiki: https://github.com/openvinotoolkit/openvino/wiki#how-to-build . Adding @ilyachur to review

Reply:
> @sgolebiewski-intel It's not really true. Basic information about the process of building OpenVINO is in the wiki: https://github.com/openvinotoolkit/openvino/wiki#how-to-build . Adding @ilyachur to review

I see. So in that case, I guess, the following article should be deleted/ moved to wiki to avoid doubling the content: 
https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/build.md

Then, the dev docs should refer to wiki, right?

Reply:
Hi @sgolebiewski-intel @helena-intel we have a plan to move build documentation from Wiki page to markdown. Jira ticket is 92089.
So https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/build.md should be the right place for building process.

Reply:
@ilyachur, @ilya-lavrenov , @helena-intel  Another suggestion of mine:
```suggestion
[Developer documentation](./docs/dev/index.md) contains information about architectural decisions which are applied inside the OpenVINO components and helps to start contribution to their development.
```
If there are plans on moving the information about building process from wiki to markdown, my previous suggestion will not stray far from the truth, will it?